src/components/LanguageButton.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import LanguageDropdown from './LanguageDropdown.js';
+import dropdown from '../assets/dropdown.png';
+
+const LanguageButton = ({ styles }) => {
+  const [style, setStyle] = useState('hidden');
+  const handleDropdown = () => {
```

Comment:
The style on the language menu is returnin a full height menu bar.
Reduce the height to fit the content. This can be achieved with display as block also. Or add h-fit` for the language bar.
Reference:
![image](https://user-images.githubusercontent.com/28519395/155631035-34e36065-f216-4ed3-93a6-af640735f12d.png)